markdown/docs/addon-author-guide.md

```diff
@@ -190,33 +193,129 @@ If your addon imports third-party code and you are unable to make
 changes to it to add Node compatibility, you can add a guard to your
 `ember-cli-build.js` file to only include it in the browser build.
 
-The FastBoot addon sets the `EMBER_CLI_FASTBOOT` environment variable
-when it is building the FastBoot version of the application. Use
-this to prevent the inclusion of a library in the FastBoot build:
+The FastBoot addon simply provides the FastBoot server a manifest of
```

Comment:
Could 'simply' be removed from this sentence without changing the overall meaning?